fv3_cap.F90

```diff
@@ -377,6 +378,15 @@ subroutine InitializeAdvertise(gcomp, rc)
                                    label ='num_files:',rc=rc)
       if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
 !
+      call ESMF_ConfigGetAttribute(config=CF,value=fv3atm_output_dir, &
```

Comment:
Do you want to put the code to module_wrt_grid_comp.F90 instead? I think only write grid comp needs to know this info.

Reply:
Done. See https://github.com/NOAA-EMC/fv3atm/pull/826/commits/975284c8343f15800b8bcfa6799e3947d2b1594a

I also updated the fms module use statements to avoid compiler warning we see in Intel builds.